environment_mace.yml

```diff
@@ -25,6 +25,8 @@ dependencies:
   - openmm-torch
   - nnpops
   - openmm-ml
+  - git
+  - openmm-ml
   - pip:
-    - ase@git+https://gitlab.com/ase/ase.git@f2615a6e9a # For PLUMED
```

Comment:
Is there any reason why to change from this commit to the main? We use specific commit to ensure that all of us use the same version of ase, as the dev version changes quickly. Alternatively, we can use version 3.23.0 if it contains all the functions we need.

Reply:
I think this version disappeared and was no longer downloadable. I will check again, but using a set version is still a good idea, i agree